ucm2/sof-soundwire/RT1308-1.conf

```diff
@@ -3,6 +3,16 @@
 SectionDevice."Speaker" {
 	Comment	"Speaker"
 
+	If.hp {
+		Condition {
+			Type ControlExists
+			Control "iface=CARD,name='Headphone Jack'"
+		}
+		True {
+			ConflictingDevice.hp "Headphone"
+		}
+	}
+
```

Comment:
Sorry no @libinyang they are not conflicting in hardware. You can perfectly play on both at the same time. This is not what Conflicting Device should be used for, we discussed with with @perexg already - or you guys changed the definition.

Reply:
I agree. Those devices are not in a conflict. @libinyang seems to try to resolve a pulseaudio behaviour (automatic switch to the given output) with this type of the UCM scheme, but I don't prefer this way, too.

Reply:
@plbossart @perexg Like @perexg said, we use this method is trying to resolve the automatic switch issue. The issue is described in https://github.com/thesofproject/linux/issues/1999. 
Sorry, I don't know that we have got the conclusion we shouldn't use Conflicting Device in this case. And I have a question: If we don't set them as conflicting devices, how can we make them playback at the same time?

Another solution is that we can use multiple profiles to fix this issue. 

btw: Currently, single profile doesn't support automatic switch in pulseaudio.

Reply:
> btw: Currently, single profile doesn't support automatic switch in pulseaudio.

This merge request is trying to resolve the port / UCM device activation in the upper layer of PA:

https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/294

Reply:
@perexg The patch https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/294 is to adjust the MIC's priority. I have tried that patch, it doesn't help on our issue.

Reply:
Yes, the implemenation is wrong. I just pointed that there's already a discussion about this problem in PA.

Reply:
@perexg Hui's patch may be only trying to adjust the priority of MIC and it may only work on the platform which have the same PCM number. On our platform, his patch doesn't work. On our platform, if we use the single profile, when we plug in the headphone, we must switch the sinks. 

btw: in the patch https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/271. I found input's profile doesn't change with the output profile. Do you have some idea for it? Thanks.